extensions/wikia/ArticleVideo/templates/ArticleVideo_featured.php

```diff
@@ -15,6 +15,7 @@
 			<div class="video-label"><?= wfMessage('articlevideo-watch')->escaped() ?><span class="video-time"></span></div>
 			<div class="video-title"></div>
 		</div>
+		<?= $app->renderView( 'ArticleVideo', 'feedback' ) ?>
```

Comment:
You can use `renderPartial` instead, that way you can get rid of empty method in controller.